src/main/java/org/broadinstitute/hellbender/cmdline/argumentcollections/IntervalArgumentCollection.java

```diff
@@ -126,13 +128,13 @@ private void parseIntervals(final GenomeLocParser genomeLocParser) {
             try {
                 includeSortedSet = IntervalUtils.loadIntervals(getIntervalStrings(), intervalSetRule, intervalMerging, intervalPadding, genomeLocParser);
             } catch( UserException.EmptyIntersection e) {
-                throw new UserException.BadArgumentValue("-L, --interval_set_rule", getIntervalStrings()+","+intervalSetRule, "The specified intervals had an empty intersection");
+                throw new CommandLineException.BadArgumentValue("-L, --interval_set_rule", getIntervalStrings()+","+intervalSetRule, "The specified intervals had an empty intersection");
             }
         }
 
         final GenomeLocSortedSet excludeSortedSet = IntervalUtils.loadIntervals(excludeIntervalStrings, IntervalSetRule.UNION, intervalMerging, intervalExclusionPadding, genomeLocParser);
         if ( excludeSortedSet.contains(GenomeLoc.UNMAPPED) ) {
-            throw new UserException("-XL unmapped is not currently supported");
+            throw new CommandLineException("-XL unmapped is not currently supported");
```

Comment:
This is changing the behaviour for exception handling, no? Before it wasn't a `CommandLineException`.